packages/medusa/src/commands/utils/get-migrations.js

```diff
@@ -101,6 +105,10 @@ export default directory => {
     return details
   })
 
+  if (!plugins) {
```

Comment:
@olivermrbl wdyt? Should we log if no plugins are provided?

Reply:
a classic scenario where this makes sense is if the user forgets to upload the keys to a Keystore or an encrypted database that is read asynchronously. , ideally you'd want to halt the server, but can/should the server run without plugins?

Reply:
At the moment it wont work without a minimum of plugins so may be it is worth throwing that some plugins are required such as payment etc? @pKorsholm wdyt?